src/nav.json

```diff
@@ -205,6 +205,10 @@
               "name": "Using custom domains",
               "slug": "custom-domains"
             },
+            {
```

Comment:
This page was originally planned to be in “platform features” since the controls are provided by the platform and this is an important concept. We don’t want to demote it to the bottom of “More guides”.

Reply:
Moved to the top of the *Implementing platform features* section.

This type of information (where it goes in the navigation) would be helpful to have in the Jira beforehand.

Reply:
You are right, sorry for the oversight. We moved this one in the reorganization and I missed it. If I missed another one, check the Content Roadmap (spreadsheet) or ask me.